examples/fluid/wetting_water_spheres_2d.jl

```diff
@@ -0,0 +1,105 @@
+# In this example two circles of water drop to the floor demonstrating the difference
+# between the behavior with and without surface tension modelling.
```

Comment:
What is this example? This comment is the same as for falling spheres 3D. Are they just using different parameters? Why do we need these separate examples?